### DISCUSSION TITLE
Collaborator permissions to make changes in the repository

### DISCUSSION THREAD

**Discussion by nGQD** [ParanoidUser/yolo#1296] [Subscribe]
Title

**Comment by ParanoidUser** [comment]
Ahoy, @nGQD! 👋

Great to hear from you! Before we continue, I'd like to learn more about your ideas. What types of changes are you thinking about?

**Reply by nGQD** [reply]
Greetings!
I originally sought write permissions to get the GitHub achievement. However, I have completed the requirements on my own repo.

Thanks for your attention by the way! Have a great day!
